website/snippets/_cloud-environments-info.md

```diff
@@ -51,33 +51,35 @@ For more info, check out this [FAQ page on this topic](/faqs/Environments/custom
 
-Extended Attributes is a text box extension at the environment level that overrides connection or environment credentials, including any custom environment variables. You can set any YAML attributes that a dbt adapter accepts in its `profiles.yml`.
-
-Something to note, Extended Attributes don't mask secret values. We recommend avoiding setting secret values to prevent visibility in the text box and logs. 
+Extended Attributes are set at the environment level, and can partially override connection or environment credentials, including any custom environment variables. You can set any YAML attributes that a dbt adapter accepts in its `profiles.yml`.
 
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/extended-attributes.jpg" width="95%" title="Extended Attributes helps users add profiles.yml attributes to dbt Cloud Environment settings using a free form text box." /> <br />
 
-If you're developing in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud), [dbt Cloud CLI](/docs/cloud/cloud-cli-installation), or [orchestrating job runs](/docs/deploy/deployments), Extended Attributes parses through the provided YAML and extracts the `profiles.yml` attributes. For each individual attribute:
-
-- If the attribute exists in another source (such as your project settings), it will replace its value (like environment-level values) in the profile. It also overrides any custom environment variables.
-
-- If the attribute doesn't exist, it will add the attribute or value pair to the profile. 
-
-Only the **top-level keys** are accepted in extended attributes. This means that if you want to change a specific sub-key value, you must provide the entire top-level key as a JSON block in your resulting YAML. For example, if you want to customize a particular field within a [service account JSON](/docs/core/connect-data-platform/bigquery-setup#service-account-json) for your BigQuery connection (like 'project_id' or 'client_email'), you need to provide an override for the entire top-level `keyfile_json` main key/attribute using extended attributes. Include the sub-fields as a nested JSON block.
-
 The following code is an example of the types of attributes you can add in the **Extended Attributes** text box:
 
 ```yaml
 dbname: jaffle_shop      
 schema: dbt_alice      
 threads: 4
+username: alice
+password: '{{ env_var(''DBT_ENV_SECRET_PASSWORD'') }}'
 ```
 
+**Extended Attributes don't mask secret values**. We recommend avoiding setting secret values to prevent visibility in the text box and logs. A common workaround is to wrap extended attributes in [environment variables](/docs/build/environment_variables), like showed above for `password: '{{ env_var(''DBT_ENV_SECRET_PASSWORD'') }}'`, that will get a value from the `DBT_ENV_SECRET_PASSWORD` environment variable at runtime.
```

Comment:
```suggestion
**Extended Attributes don't mask secret values**. We recommend avoiding setting secret values to prevent visibility in the text box and logs. A common workaround is to wrap extended attributes in [environment variables](/docs/build/environment-variables), like showed above for `password: '{{ env_var(''DBT_ENV_SECRET_PASSWORD'') }}'`, that will get a value from the `DBT_ENV_SECRET_PASSWORD` environment variable at runtime.
```